app/marketing/management/commands/sync_pull_request_with_bounty_fulfillments.py

```diff
@@ -0,0 +1,138 @@
+'''
+    Copyright (C) 2017 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+'''
+import logging
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from dashboard.models import BountyFulfillment
+from dashboard.utils import record_funder_inaction_on_fulfillment
+from git.utils import get_gh_issue_state, get_interested_actions, post_issue_comment
+from marketing.mails import funder_payout_reminder
+
+logger = logging.getLogger('')
+console = logging.StreamHandler()
+console.setLevel(logging.INFO)
+console.setFormatter(logging.Formatter('%name)s - %(levelname)s - %(message)s'))
+logger.addHandler(console)
+
+
+class Command(BaseCommand):
```

Comment:
Would definitely love some tests around this management command :+1:

Reply:
@danlipert Just added a test to the management command, please review